<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>716</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLineEdit" name="vrem">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>152</width>
     <height>185</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="rach">
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>расчёска</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="puz">
      <property name="text">
       <string>пузырёк</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="bistr">
      <property name="text">
       <string>быстрая</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="gn">
      <property name="text">
       <string>гномья</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="manky">
      <property name="text">
       <string>обезьянья</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>наносекунд</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tip">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>530</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="mist">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>310</y>
     <width>611</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>550</y>
     <width>311</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Найдено значений</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="chsl">
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>530</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>тип поиска</string>
   </property>
  </widget>
  <widget class="QPushButton" name="min">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QPushButton" name="max">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>max</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="max1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>211</width>
     <height>251</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <row/>
   <column>
    <property name="text">
     <string>1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="kn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>161</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>отсортировать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="dub">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>161</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>удалить дубликаты</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="min1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="po">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>420</y>
     <width>311</width>
     <height>107</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>230</y>
     <width>68</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>время</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000000</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="ch">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>374</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="random">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>161</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>random</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kn2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>искать</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>251</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>введите число для поиска</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sr1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sr">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ср.знач</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
